README.md

```diff
@@ -9,22 +9,56 @@
 [![build](https://img.shields.io/github/workflow/status/exadel-inc/esl/validate/main?style=for-the-badge)](https://github.com/exadel-inc/esl/actions/workflows/validate.yml)
 [![License](https://img.shields.io/badge/license-MIT-green.svg?style=for-the-badge)](./README.md)
 
-Exadel Smart Library (**ESL**) is a web components based library that gives you a set of **lightweight** and **flexible** custom elements to easily create basic UX modules and make your websites super fast.
+Exadel Smart Library (**ESL**) is an open-source **lightweight** and **flexible** UI component library based on web components.
```

Comment:
I suggest write "UI components (plural form) library" throughout the document.

Reply:
Have confirmation from the SEO team that's the form of keyword important here and unfortunately
"UI component library" is a popular keyword but the "UI components library" is not (